Carnap-Server/Foundation.hs

```diff
@@ -295,20 +366,20 @@ instance YesodAuth App where
                                    (Just (User ident _), Nothing) ->  redirect (RegisterR ident)
                                    (Nothing,_) -> return ()
                         --if so, go ahead
-                        Just ud -> setMessage "Now logged in"
+                        Just _ -> setMessage "Now logged in"
 
     -- appDevel is a custom method added to the settings, which is true
     -- when yesod is running in the development environment and false
     -- otherwise
     authPlugins app = let settings = appSettings app in
                           if appDevel settings
-                              then [ authDummy ]
-                              else [ oauth2GoogleScoped ["email","profile"] (appKey settings) (appSecret settings) ]
+                              then [ authDummy, authLTI13 ]
```

Comment:
It looks like lti13 might need a link for the AuthPlugin login? 

https://github.com/lf-/lti13/blob/c0aad6339c767d6c8e1780288da609915a87a89d/yesod-auth-lti13/src/Yesod/Auth/LTI13.hs#L325

At the moment, I'm not sure how to click through to attempt authentication there.

Reply:
This is actually bad copy on my part, not a technical error, but if you hit it, someone else definitely will too. It's just a note that you can go to your learning management system to initiate login. If you need a sample learning management system, there's the reference implementation, which there are details for in the first post (which you can use directly yourself; there's no login on it or anything). You can add the configuration details on the admin page.

Reply:
As discussed on Matrix, this is resolved with both the addition of `authLTI13WithWidget` as of yesod-auth-lti13-0.1.2.0, and I've also attempted to improve the copy independently.

Reply:
Should we perhaps upgrade the deps on this PR, and use `authLTI13WithWidget`, rather than `authLTI13` in production?